src/Syntax.jl

```diff
@@ -158,6 +159,48 @@ macro stock_and_flow(block)
     end
 end
 
+abstract type StockArgT end
+abstract type ParamArgT end
+
+# struct StockArgUnitless <: StockArgT
```

Comment:
Why are these (and the `convert`s below) commented out?

Reply:
They don't need to be there at all.  Was originally going to use Unitless when, well, it didn't have a unit, but that was folded into Symbol with :NONE.  Converts were going to be helper functions for creating them, but proved unnecessary.  They can be deleted.